renovate.config.js

```diff
@@ -0,0 +1,57 @@
+module.exports = (config) => {
+	if (!config.isSelfHosted) {
+	  console.log('Renovate is disabled when running via GitHub App.');
+	  return {
+		enabled: false,
+		onboarding: false,
+	  };
+	}
```

Comment:
Can this condition occur?

Reply:
Yes, this is to stop the hosted version of Renovate from running.  As there is no "Exclude repo" option in Github, our only options are to have the Hosted version of renovate Org wide, or select every repo manually.